testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suspendresume/DomainGracefulShutdownTestCase.java

```diff
@@ -139,96 +168,146 @@ public Object call() throws Exception {
 
 
         } finally {
-            plan = deploymentManager.newDeploymentPlan().undeploy(WEB_SUSPEND_JAR)
-                    .andRemoveUndeployed()
-                    .toServerGroup(MAIN_SERVER_GROUP)
-                    .build();
-            deploymentManager.execute(plan).get();
+            executorService.shutdown();
         }
-
-
     }
 
     @Test
     public void testStartSuspendedDomainMode() throws Exception {
-
         DomainClient client = domainMasterLifecycleUtil.getDomainClient();
-        DomainDeploymentManager deploymentManager = client.getDeploymentManager();
-        DeploymentPlan plan = deploymentManager.newDeploymentPlan().add(WEB_SUSPEND_JAR, createDeployment().as(ZipExporter.class).exportAsInputStream())
-                .andDeploy().toServerGroup(MAIN_SERVER_GROUP)
-                .build();
-        deploymentManager.execute(plan).get();
 
+        ModelNode op = new ModelNode();
+        op.get(ModelDescriptionConstants.OP).set("reload-servers");
+        op.get(ModelDescriptionConstants.TIMEOUT).set(60);
```

Comment:
> Don't we want to use suspend-timeout rather than now-deprecated timeout attribute? 

My intention was not to change existing tests to use the new attribute rename since TIMEOUT, although it is deprecated, must be still valid. From my POV, I see it like a more convenient task to do when we definely remove the TIMEOUT attribute. @jstourac Do you agree?

> Actually, looking on the description of the :reload-servers() operation, I don't see there is any timeout or suspend-timeout attribute available? As described in the dev-analysis, one should use :suspend-servers() operation first in such case?

Yes, correct. It looks like the TIMEOUT was wrongly added there for this operation, it is an old test. I remove it since it is not used.